server/controllers/amp-page.js

```diff
@@ -71,6 +71,7 @@ function getAndRender(uuid, options) {
 
 			const shareParams = {};
 			article.shareUrl = `${article.webUrl}?${querystring.stringify(shareParams)}`;
+			article.facebookAppId = '121355018262104';
```

Comment:
@georgecrawford can you think of any reason not to use the FT Instant Articles app id?